app/org/sagebionetworks/bridge/models/accounts/GenericAccount.java

```diff
@@ -143,6 +146,51 @@ public void setEmail(String email) {
 
     /** {@inheritDoc} */
     @Override
+    public Phone getPhone() {
+        return phone;
+    }
+
+    /** {@inheritDoc} */
+    @Override
+    public void setPhone(Phone phone) {
+        this.phone = phone;
+    }
+    
+    /** {@inheritDoc} */
+    @Override
+    public Boolean getEmailVerified() {
+        // For accounts prior to the introduction of the email/phone verification flags, where 
+        // the flag was not set on creation or verification of the email address, return the right value.
+        if (emailVerified == null) {
+            if (status == AccountStatus.ENABLED) {
+                return Boolean.TRUE;
+            } else if (status == AccountStatus.UNVERIFIED) {
+                return Boolean.FALSE;
+            }
+        }
```

Comment:
Similarly, if we moved this logic to the unmarshall() code, we wouldn't have to write or test this twice.